testing/wslu/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: Patrick Wu <wotingwu@live.com>
+# Contributor: Patrick Wu <wotingwu@live.com>
+pkgname=wslu
+pkgver=2.3.2
+pkgrel=0
+pkgdesc="A collection of utilities for Windows 10 Linux Subsystems"
+url="https://github.com/wslutilities/wslu"
+arch="noarch"
+license="GPL-3.0-or-later"
+depends="bc bash-completion imagemagick"
+source="$pkgname-$pkgver.tar.gz::https://github.com/wslutilities/wslu/archive/v$pkgver.tar.gz"
+options="!check" #tests can be only run on a distro running on WSL(Windows Subsystems for Linux) environment, hence disabled
+
+build() {
+	make
+}
+
+check() {
+	make test
+}
+
+doc() {
+	install -Dm 555 docs/* -t "$pkgdir"/usr/share/man/man1/
+}
```

Comment:
Just keep that step in `package()` and add `subpackages="$pkgname-doc"` to the APKBUILD.